listenbrainz/db/stats.py

```diff
@@ -38,6 +36,11 @@
 from pydantic import ValidationError
 
 
+# sitewide statistics are stored in the user statistics table
+# as statistics for a special user with the following user_id
+SITEWIDE_STATS_USER_ID = 15753
```

Comment:
Should this perhaps be user 1 -- the Anonymous user in musicbrainz for various purposes?

Reply:
This is the ListenBrainz User ID, not the musicbrainz row id. Currently, LB user id `1` is in use.

Reply:
Ah, I see.